packages/qri-node/lib/qri-node.js

```diff
@@ -1,25 +1,41 @@
 const fs = require('fs-extra')
-const { spawn } = require('child_process');
+const { spawnSync } = require('child_process');
 
 module.exports = {
+  /**
+  * 'initialize' creates a new dataset, links it to the current directory,
```

Comment:
'initialize' -> 'init', since that's the command's real name.

Reply:
👍